eva-accession-release-automation/gather_clustering_counts/tests/test_gather_release_counts.py

```diff
@@ -1,4 +1,12 @@
-from gather_clustering_counts.gather_release_counts import find_link
+import os
+from unittest import TestCase
+from unittest.mock import patch
+
+from sqlalchemy import select
+from sqlalchemy.orm import Session
+
+from gather_clustering_counts.gather_release_counts import find_link, ReleaseCounter
+from gather_clustering_counts.release_count_models import RSCountPerTaxonomy
```

Comment:
I can't find `RSCountPerTaxonomy` anywhere, did something not get pushed or am I just missing it?

Reply:
Sorry. You're correct. I forgot to push ...